cmd/kubelet/app/options/container_runtime.go

```diff
@@ -26,9 +26,9 @@ import (
 )
 
 const (
-	// When these values are updated, also update test/e2e/framework/util.go
+	// When these values are updated, also update test/utils/image/manifest.go
```

Comment:
These "please also update" comments in multiple files seem a bit too much to me. Maybe we should clean it up another time